GameDataParser/Files/SetPath.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace GameDataParser.Files
+{
+    public class SetPath
+    {
+        public const string XML_PATH = @"H:\MS2 Server\GMS2_final_version\maplestory2\appdata\Data\Xml.m2d";
```

Comment:
is this going to work for other people?

Reply:
For other people to work they have to set this manually to their actual path of the game, i can add a comment to let people know what file needs on those path

Reply:
We really want to avoid putting in hardcoded absolute paths within the source code. If anything, at least include a relative path that people can put into their repo folder and add that folder to the git exceptions list.